person.rb

```diff
@@ -1,6 +1,6 @@
 require './corrector'
 class Person
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rents
   attr_reader :id
 
```

Comment:
- [ ] The `Person` class should have a method for adding rentals, it may look like this :point_down: 

```ruby
def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```